src/AlphaHomoraV2/mappings/PoolRegistry.ts

```diff
@@ -0,0 +1,343 @@
+import { Address, BigInt, log } from "@graphprotocol/graph-ts";
+import { OwnershipTransferred } from "../../../generated/AlphaHomoraV2PoolRegistry/AlphaHomoraV2PoolRegistry";
+import { AlphaHomoraV2ERC20 } from "../../../generated/AlphaHomoraV2PoolRegistry/AlphaHomoraV2ERC20";
+import { AlphaHomoraV2CyToken } from "../../../generated/AlphaHomoraV2PoolRegistry/AlphaHomoraV2CyToken";
+import { convertToLowerCase } from "../../utils/converters";
+import {
+  AlphaHomoraV2WMasterChef,
+  AlphaHomoraV2Token,
+  AlphaHomoraV2LpPair,
+  AlphaHomoraV2RewardToken,
+  AlphaHomoraV2CreamToken,
+} from "../../../generated/schema";
+
+/*
+Array ordered as:[
+    [pool key, wMasterChefAddress, lpTokenAddress, name, pid, chefAddress, exchange, token0, token1, rewardtoken0, rewardToken1(optional)],...
+]
+*/
+
+export const poolRegistry: Array<string[]> = [
```

Comment:
Where is this coming from? Can't we get this via web3 calls (or by listening to some events)?
The web site lists 18 pools, but here are only 16.

Reply:
will commit change to this static list soon, going to make it dynamic